source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -688,5 +688,41 @@ than an element in the range being searched.
 
 The elements e of [start, end) must be partitioned with respect to the comparator used.
 
+.. code:: cpp
+
+  template <typename Policy, typename InputIt, typename OutputIt, typename UnaryOp,
+      typename UnaryPredicate>
+  OutputIt
+  transform_if(Policy&& policy, InputIt start, InputIt end, OutputIt result, UnaryOp op,
+      UnaryPredicate pred);                                                               // (1)
+
+  template <typename Policy, typename InputIt1, typename InputIt2, typename OutputIt, 
+      typename BinaryOp, typename BinaryPredicate>
+  OutputIt
+  transform_if(Policy&& policy, InputIt1 start1, InputIt1 end1, InputIt2 start2, OutputIt result,
+      BinaryOp op, BinaryPredicate pred);                                                 // (2)
+
+:code:`oneapi::dpl::transform_if` applies a given function to the elements of the input sequence(s) that
+satisfy a given predicate, and stores the result to the output. Depending on the arguments, the algorithm
+
+1.
+evaluates the unary predicate :code:`pred` for each position :code:`i` of the sequence
+:code:`[start, end)` and if :code:`pred(start[i]) == true`, it performs the unary operation
+:code:`op(start[i])` and stores the result into :code:`result[i]`. If
```

Comment:
I would change the formatting to create a numbered list:
```
1. evaluates the unary predicate :code:`pred` for each position :code:`i` of the sequence
   :code:`[start, end)` and if :code:`pred(start[i]) == true`, it performs the unary operation
   :code:`op(start[i])` and stores the result into :code:`result[i]`. If
```
etc.

Reply:
I believe it is fixed now, at least in the github rendering and as I understand the rst syntax.  However, it might be good to get @dcbenito to take a look, I will reach out.